sphinx-markdown-2021.rst

```diff
@@ -0,0 +1,115 @@
+.. post:: Dec 16, 2021
+   :tags: media, talk, conference, markdown, commonmark
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      During 2021 we have promoted the use of Markdown in Sphinx in various ways.
+      In this post we show how you can start using it in your project today,
+      and give some extra pointers.
+
+Sphinx and Markdown around the world in 2021
+============================================
+
+Read the Docs has been committed to improving the accessibility
+and user experience of Sphinx since its beginnings,
+and that includes the markup language in which the documentation is written.
+After carefully listening to users
+that wanted to use Markdown in Sphinx as well as reStructuredText,
+:doc:`we announced recommonmark more than six years ago </adding-markdown-support>`
+to help bridging the immense popularity of Markdown
+with the powerful capabilities of Sphinx.
+
+It is no surprise that Markdown is so demanded:
+thanks in large part to the huge popularity of GitHub,
+`Markdown is nowadays the most widely used markup language in open-source
+projects <https://passo.uno/docs-as-code-tools-open-standards/>`_,
+ahead of reStructuredText, the second in the list, by an order of magnitude.
+
+This year, thanks to the CZI Grant we received last year
+and the effort of other communities,
+especially `the Executable Book Project <https://executablebooks.org>`_,
+we have devoted lots of resources to help consolidate Markdown
+as a compelling markup language for Sphinx.
+
+Writing Markdown in Sphinx
+--------------------------
+
+To that end, we recognized the potential of MyST as a successor of recommonmark,
+and :doc:`deprecated the latter in favor of MyST-Parser </newsletter-april-2021>`.
+To :doc:`start writing Markdown in Sphinx <sphinx:usage/markdown>`,
+you need to follow three steps:
+
+* Install ``MyST-Parser``: ``pip install myst-parser``
+
+* Add ``myst_parser`` to the list of Sphinx extensions:
+
+.. code-block:: python
+   :emphasize-lines: 3
+
+   extensions = [
+       # ...other extensions
+       "myst_parser",
+   ]
+
+* Start writing content in ``*.md`` files!
+
+You can find more information in the :doc:`MyST-Parser documentation <myst-parser:index>`.
+
+Talks and workshops at conferences
+----------------------------------
+
+Luckily the migration to MyST-Parser was trivial for most projects,
+but we still wanted to spread the word about
+the best practices for creating Sphinx projects on Read the Docs using MyST Markdown.
```

Comment:
Now, after reading this paragraph, I seems that migrating users from recommonmark to MyST was part of this job and now, having a specific section commenting that recommonmark was deprecated in favor or MyST makes more sense.

Reply:
Since you made the earlier comments before reading the whole post, I prefer to wait on @ericholscher to do the proofreading. Thanks anyway @humitos !